src/components/ChatBody/ChatBody.js

```diff
@@ -60,15 +61,53 @@ const ChatBody = ({ height, anonymousMode, showRoles, GOOGLE_CLIENT_ID }) => {
     return () => RCInstance.close();
   }, [isUserAuthenticated, getMessagesAndRoles]);
 
+  const [onDrag, setOnDrag] = useState(false);
+  const [leaveCount, setLeaveCount] = useState(0);
+
+  const handleDrag = (e) => {
+    e.preventDefault();
+  };
+
+  const handleDragEnter = () => {
+    setOnDrag(true);
+  };
+  const handleDragLeave = () => {
+    if (leaveCount % 2 === 1) {
+      setOnDrag(false);
+      setLeaveCount(leaveCount + 1);
+    } else {
+      setLeaveCount(leaveCount + 1);
+    }
+  };
+
+  const handleDragDrop = (e) => {
+    e.preventDefault();
+    setOnDrag(false);
+    setLeaveCount(0);
+    RCInstance.sendAttachment(e.dataTransfer.files[0]);
+  };
+
   return (
     <Box
       style={{
         borderLeft: '1px solid #b1b1b1',
         borderRight: '1px solid #b1b1b1',
+        paddingTop: '70px',
       }}
+      onDragOver={(e) => handleDrag(e)}
+      onDragEnter={handleDragEnter}
+      onDragLeave={handleDragLeave}
       className={styles.container}
       height={height}
     >
+      {onDrag ? (
+        <Box
+          onDrop={(e) => handleDragDrop(e)}
+          className={styles.drag_component}
+        >
+          DRAG AND DROP AREA
```

Comment:
can we please change it to - `Drop to upload file` - same as RC

Reply:
Sure sir 👍

Reply:
Done 👍